.gitignore

```diff
@@ -58,3 +58,6 @@ typings/
 .env
 
 build/
+
+# package lock
+package-lock.json
```

Comment:
we want to commit the package-lock, I think. https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5